packages/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -256,13 +256,19 @@ export class Tabs extends React.Component<TabsProps, TabsState> {
                   id: childId,
                   tabContentId,
                   isHidden = false,
+                  className: childClassName = '',
```

Comment:
Can you try just using classname here.  It looks like `className` was not destructured before and that might have been the issue.  I don't think you need to rename it to `childClassName`.

Reply:
I was afraid of `className` in line 197 shadowing this `className`. But you are probably right.

Reply:
Well, nope - it didn't work. eslint is not happy:

```
/home/circleci/project/packages/react-core/src/components/Tabs/Tabs.tsx
  259:19  error  'className' is already declared in the upper scope                                                                                                                                                                                                                      no-shadow
  266:36  error  Replace `⏎······················styles.tabsItem,⏎······················eventKey·===·activeKey·&&·styles.modifiers.current,⏎······················className⏎····················` with `styles.tabsItem,·eventKey·===·activeKey·&&·styles.modifiers.current,·className`  prettier/prettier
```